site/gatsby-site/src/components/reports/ReportCard.js

```diff
@@ -76,11 +73,11 @@ const ReportCard = ({ item, className = '', incidentId }) => {
           onClick={(e) => e.stopPropagation()}
           onKeyDown={(e) => e.stopPropagation()}
         >
-          <a href={`#r${item.report_number}`} className="max-w-full cursor-pointer">
-            <h5 className="text-xl font-bold tracking-tight text-gray-900 dark:text-white w-full hover:text-primary-blue">
+          <div role="presentation" onClick={toggleReadMore} onKeyDown={toggleReadMoreKeyDown}>
```

Comment:
I'm guessing that the `role="presentation"` is there because the linter complained about having event handlers on a div with no aria role. This silences the warning, but it doesn't solve the problem that the element can't be focused in keyboard or screen-reader navigation. It's not a big deal since there's another button that does the same thing at the bottom, but to be semantically correct I'd prefer if this were a `<button>` styled as ordinary text.

Also, I noticed now that cursor switches to a pointer when hovering over the text, even though nothing happens when you click there. If possible, it would be best to the cursor to a pointer only when it's the margins, which can be clicked.

![image](https://user-images.githubusercontent.com/25443411/223872474-9725391b-d678-4d78-8aee-1ae14435bb3a.png)